packages/astro/src/@types/astro.ts

```diff
@@ -2356,7 +2360,7 @@ export interface AstroUserConfig {
 		 *     const blog = defineCollection({
 		 *       // For content layer you no longer define a `type`
 		 *      type: 'content',
-		 *      loader: glob({ pattern: "**\/*.md", base: "./src/data/blog" }),
+		 *      loader: glob({ pattern: '**/[^_]*.(md|mdx)', base: "./src/data/blog" }),
```

Comment:
```suggestion
		 *      loader: glob({ pattern: '**/[^_]*.md', base: "./src/data/blog" }),
```
This is simpler, as it's just `md`